databricks_blueprints/delete_file_from_dbfs.py

```diff
@@ -0,0 +1,65 @@
+"""
+
+"""
+import argparse
+import sys
+import requests
+import shipyard_utils as shipyard
+try:
+    import helpers
+    import errors
+except BaseException:
+    from . import helpers
+    from . import errors
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--instance-id', dest='instance_id', required=True)
+    parser.add_argument('--source-file-name', dest='source_file_name', required=True)
+    parser.add_argument('--source-folder-name', dest='source_folder_name', required=True)
+    args = parser.parse_args()
+    return args
+
+
+def delete_file_from_dbfs(client, file_path_and_name):
+    delete_endpoint = "/dbfs/delete"
+    data = {
+        'path': file_path_and_name
+    }
+    print(f"Start delete for {file_path_and_name}")
+    delete_response = client.post(delete_endpoint, data=data)
+    if delete_response.status_code == requests.codes.ok:
+        print(f"DBFS File: {file_path_and_name} delete function completed successfully")
```

Comment:
Returns success message even if the file did not exist. Should catch for that and return separate error.